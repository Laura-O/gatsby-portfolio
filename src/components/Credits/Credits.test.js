import React from 'react';
import { shallow } from 'enzyme';
import Imprint from './Imprint';

test('rendering', () => {
  const wrapper = shallow(<Imprint path="/something" />);
  expect(wrapper).toMatchSnapshot();
});
