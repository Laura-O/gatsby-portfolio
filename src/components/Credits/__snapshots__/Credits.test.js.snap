// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Credits
    path="/something"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div>
        <h3>
          Credits
        </h3>
        Icons made by
         
        <a
          href="https://www.flaticon.com/authors/smashicons"
        >
          Smashicons
        </a>
         and
         
        <a
          href="https://www.flaticon.com/authors/swifticons"
        >
          Swifticons
        </a>
        from 
        <a
          href="https://www.flaticon.com"
        >
          www.flaticon.com
        </a>
      </div>,
      "className": "credits",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h3>
            Credits
          </h3>,
          "Icons made by",
          " ",
          <a
            href="https://www.flaticon.com/authors/smashicons"
          >
            Smashicons
          </a>,
          " and",
          " ",
          <a
            href="https://www.flaticon.com/authors/swifticons"
          >
            Swifticons
          </a>,
          "from ",
          <a
            href="https://www.flaticon.com"
          >
            www.flaticon.com
          </a>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Credits",
          },
          "ref": null,
          "rendered": "Credits",
          "type": "h3",
        },
        "Icons made by",
        " ",
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Smashicons",
            "href": "https://www.flaticon.com/authors/smashicons",
          },
          "ref": null,
          "rendered": "Smashicons",
          "type": "a",
        },
        " and",
        " ",
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Swifticons",
            "href": "https://www.flaticon.com/authors/swifticons",
          },
          "ref": null,
          "rendered": "Swifticons",
          "type": "a",
        },
        "from ",
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "www.flaticon.com",
            "href": "https://www.flaticon.com",
          },
          "ref": null,
          "rendered": "www.flaticon.com",
          "type": "a",
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div>
          <h3>
            Credits
          </h3>
          Icons made by
           
          <a
            href="https://www.flaticon.com/authors/smashicons"
          >
            Smashicons
          </a>
           and
           
          <a
            href="https://www.flaticon.com/authors/swifticons"
          >
            Swifticons
          </a>
          from 
          <a
            href="https://www.flaticon.com"
          >
            www.flaticon.com
          </a>
        </div>,
        "className": "credits",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h3>
              Credits
            </h3>,
            "Icons made by",
            " ",
            <a
              href="https://www.flaticon.com/authors/smashicons"
            >
              Smashicons
            </a>,
            " and",
            " ",
            <a
              href="https://www.flaticon.com/authors/swifticons"
            >
              Swifticons
            </a>,
            "from ",
            <a
              href="https://www.flaticon.com"
            >
              www.flaticon.com
            </a>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Credits",
            },
            "ref": null,
            "rendered": "Credits",
            "type": "h3",
          },
          "Icons made by",
          " ",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Smashicons",
              "href": "https://www.flaticon.com/authors/smashicons",
            },
            "ref": null,
            "rendered": "Smashicons",
            "type": "a",
          },
          " and",
          " ",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Swifticons",
              "href": "https://www.flaticon.com/authors/swifticons",
            },
            "ref": null,
            "rendered": "Swifticons",
            "type": "a",
          },
          "from ",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "www.flaticon.com",
              "href": "https://www.flaticon.com",
            },
            "ref": null,
            "rendered": "www.flaticon.com",
            "type": "a",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
